e2e/framework/README.md

```diff
@@ -0,0 +1,23 @@
+# rook-test-framework
```

Comment:
for easier visibility, i'm thinking this README should be under the `e2e` folder, not down under `e2e/framework`.